projects/plugins/jetpack/modules/.eslintrc.js

```diff
@@ -1,39 +1,31 @@
 module.exports = {
 	root: true,
-	plugins: [ 'es5' ],
-	extends: [ 'eslint:recommended', 'plugin:es5/no-es2015' ],
+	extends: [ 'eslint:recommended' ],
 	env: {
 		browser: true,
-		jquery: true,
+		es6: true,
 	},
 	globals: {
-		_: false,
```

Comment:
these three lines need to remain.

Reply:
Internal ref: p1636751680290600-slack-CBG1CP4EN